### DISCUSSION TITLE
Getting an attribute error

### DISCUSSION THREAD

**Discussion by nikhileshwani** [hbldh/bleak#1305] [Q&A]
Hello, 

I am running the uart_service.py file with my bluetooth device. The code is giving me an attribute error
AttributeError: 'StdInputFile' object has no attribute 'buffer'

Please help with the same 

Thanks in advance!

Regards, 
Nikhilesh

**Comment by dlech** [comment]
Can you share the complete stack trace of the error? What operating system are you using? How did you run the program?
